reactive/kubernetes_control_plane.py

```diff
@@ -237,6 +238,10 @@
     when_not="ceph-client.available",
     clear_flag="kubernetes-control-plane.apiserver.configured",
 )
+# when CNI becomes available, reconfigure k8s services with the cluster-cidr
+register_trigger(
+    when="cni.available", clear_flag="kubernetes-control-plane.components.started"
+)
```

Comment:
This tripped me up for a minute because we have a `when_not` trigger against `cni.available` above. Then i remembered/saw that triggers run once per endpoint, so these flags won't bounce around during every reactive run.